jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -670,7 +652,8 @@ private void startIce4j(
             }
 
             // Reload for all the new system properties to be seen
-            JitsiConfig.Companion.reload();
+            // TODO(brian): I don't think this reload is necessary?
+//            JitsiConfig.Companion.reload();
```

Comment:
Figure this out, and remove it if it's really not necessary?

Reply:
Yeah, still need to verify that.